helm/superset/README.md

```diff
@@ -23,7 +23,9 @@ NOTE: This file is generated by helm-docs: https://github.com/norwoodj/helm-docs
 
 # superset
 
-![Version: 0.8.5](https://img.shields.io/badge/Version-0.8.5-informational?style=flat-square)
+
+
```

Comment:
Hi @rajan123456! Remove these 2 line and I guess `helm-docs` will be fixed!

Reply:
Thanks. On my system, regenerating the helm-docs automatically adds these lines back, not sure why. I removed them manually and pushed it again. Hope CI is ok now. Thanks!